packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -575,7 +581,9 @@ export class VaFileInput {
                 {(file || value || uploadedFile) && (
                   <div>
                     {this.showSeparator && <hr class="separator" />}
-
+                    {encrypted && (
+                      <va-text-input label="File password" required />
```

Comment:
`va-text-input` doesn't support type `password`. I wonder if this will be an issue down the road?

Reply:
I was wondering why the fields were not password fields :) Seems like that should be an option for va-text-input